### DISCUSSION TITLE
Toolbar Button Highlight

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#4411] [Q&A]
Hi. 
I wanted to know if there is any way to highlight an active button in the toolbar. Currently, we are changing the icon to show that this button is currently active. We want to just be able to highlight it, as it is simpler to scale.

Thank You.

**Comment by NancyMcCallB** [comment]
Hi, @hanan-commits-work,

Are your items defined as ActionItemButtons or something else? There's an isActive member of the ActionItemButton state that determines whether the active item indicator is drawn for that button.

Thanks,
Nancy McCall
